bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java

```diff
@@ -228,29 +233,13 @@ else if (Boolean.class.isAssignableFrom(option.getType()))
         FileUtils.deleteQuietly(windupConfiguration.getOutputDirectory().toFile());
         Path graphPath = windupConfiguration.getOutputDirectory().resolve(GRAPH_DATA_SUBDIR);
 
-        System.out.println();
-        if (windupConfiguration.getInputPaths().size() == 1)
-        {
-            System.out.println("Input Application:" + windupConfiguration.getInputPaths().iterator().next());
-        }
-        else
-        {
-            System.out.println("Input Applications:");
-            for (Path inputPath : windupConfiguration.getInputPaths())
-            {
-                System.out.println("\t" + inputPath);
-            }
-            System.out.println();
-        }
-        System.out.println("Output Path:" + windupConfiguration.getOutputDirectory());
-        System.out.println();
-
```

Comment:
Should this be pronted in RunWindupCommand? I think it should rather be printed in WindupProcessorImpl so it's printed in every execution path.